hphp/test/slow/ext_file/empty_filename.php.expectregex

```diff
@@ -0,0 +1,20 @@
+Warning\: chmod\(\)\: No such file or directory in [^\r\n]+\/test\/slow\/ext_file\/empty_filename.php on line 2
+bool\(false\)
+bool\(false\)
+
+Warning\: disk_free_space\(\)\: (No such file or directory|Operation not permitted) in [^\r\n]+\/test\/slow\/ext_file\/empty_filename.php on line 4
+bool\(false\)
+
+Warning\: chmod\(\)\: No such file or directory in [^\r\n]+\/test\/slow\/ext_file\/empty_filename.php on line 5
+bool\(false\)
+bool\(false\)
+
+Warning\: disk_free_space\(\)\: (No such file or directory|Operation not permitted) in [^\r\n]+\/test\/slow\/ext_file\/empty_filename.php on line 7
+bool\(false\)
+
+Warning\: chmod\(\)\: No such file or directory in [^\r\n]+\/test\/slow\/ext_file\/empty_filename.php on line 8
+bool\(false\)
+bool\(false\)
+
+Warning\: disk_free_space\(\)\: Operation not permitted in [^\r\n]+\/test\/slow\/ext_file\/empty_filename.php on line 10
```

Comment:
@cmuellner If this was optional, like line 12, I think this would be good to land into master.